ignite/contrib/handlers/tqdm_logger.py

```diff
@@ -10,47 +10,84 @@ class ProgressBar:
     """
     TQDM progress bar handler to log training progress and computed metrics.
 
+    Args:
+        persist (bool): (Optional) set to ``True`` to persist the progress bar after completion (default = ``False``)
+
     Examples:
 
-        Create a progress bar that shows you some metrics as they are computed,
-        by simply attaching the progress bar object to your engine.
+        Simple progress bar
+
+        .. code-block:: python
+
+            trainer = create_supervised_trainer(model, optimizer, loss)
+
+            pbar = ProgressBar()
+            pbar.attach(trainer)
+
+        Attach metrics that already has been computed (such as `RunningAverage`)
```

Comment:
```suggestion
        Attach metrics that already have been computed at `ITERATION_COMPLETED` (such as `RunningAverage`)
```

This better describes it IMO, as the metric isn't already computed at attach time, but should have updated the `state.metrics` of the engine before the bar updates, at ITERATION_COMPLETED

Reply:
just fixed that :) thanks!